src/codegen/generate.ts

```diff
@@ -364,15 +363,6 @@ export default class ApiGenerator {
   getRefAlias(obj: OpenAPIV3.ReferenceObject, onlyMode?: OnlyMode) {
```

Comment:
The changes in this method essentially make it so that we always create all possible permutations of a referenced type (base/read/write).
Prior to this we were a little more regressive and only created the read/write types when the ref actually was used in a context where they would be needed.